SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -1873,126 +1900,38 @@ void CNEMOEulerSolver::SetNondimensionalization(CConfig *config, unsigned short
 
 void CNEMOEulerSolver::BC_Sym_Plane(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics,
                                     CNumerics *visc_numerics, CConfig *config, unsigned short val_marker) {
-
-  unsigned short iDim, jDim, iSpecies, iVar, jVar;
   unsigned long iPoint, iVertex;
+  unsigned short iDim, iVar;
+  su2double Area, Normal[3], UnitNormal[3];
+  const su2double* Residual_Old;
+  su2double *Residual = new su2double[nVar];
 
-  su2double *Normal = nullptr, Area, UnitNormal[3], *NormalArea,
-  **Jacobian_b, **DubDu,
-  rho, cs, P, rhoE, rhoEve, conc, *u, *dPdU;
+  BC_Euler_Wall(geometry, solver_container, conv_numerics, visc_numerics, config, val_marker);
```

Comment:
> Euler boundary condition is now called inside the symmetry plane boundary condition, before performing the residual correction
This way, I believe I am assuring for all the vector quantities to be parallel to the symmetry plane and guaranteeing the condition of symmetry according to Jiri Blazek's - "Computational fluid dynamics: principles and applications" . However, calling Euler function does not appear to substantially change the the solution substantially in the test I have made so far.

Imo this is not necessary and that part can be deleted (until Jiri's suggested implementation shows problems).